arangod/Pregel/Utils.cpp

```diff
@@ -71,7 +71,7 @@ ErrorCode Utils::resolveShard(ClusterInfo& ci, WorkerConfig const* config,
   std::shared_ptr<LogicalCollection> info;
   auto const& it = planIDMap.find(collectionName);
   if (it != planIDMap.end()) {
-    info = ci.getCollectionNT(config->database(), it->second);  // might throw
+    info = ci.getCollectionNT(config->database(), it->second);
```

Comment:
We should make a TODO card to (re)move `resolveShard` from Pregel. This should have no business being here ;)

Reply:
https://arangodb.atlassian.net/browse/PREG-206